daemon/pod.go

```diff
@@ -407,16 +407,7 @@ func (p *Pod) init(data interface{}) error {
 		return err
 	}
 
-	resPath := filepath.Join(DefaultResourcePath, p.Id)
-	if err := os.MkdirAll(resPath, os.FileMode(0755)); err != nil {
-		glog.Error("cannot create resource dir ", resPath)
-		return err
-	}
-
-	status := hypervisor.NewPod(p.Id, p.spec, &hypervisor.HandleEvent{hyperHandlePodEvent, data})
-	status.ResourcePath = resPath
-
-	p.status = status
+	p.status = hypervisor.NewPod(p.Id, p.spec, &hypervisor.HandleEvent{hyperHandlePodEvent, data})
```

Comment:
The definition is in runV project, will submit the PR to remove it after this is merged